app.js

```diff
@@ -119,7 +119,8 @@ app.get('/compile', async (req, res) => {
         preparation = await latexOnline.prepareURLCompilation(req.query.url, command);
     } else if (req.query.git) {
         var workdir = req.query.workdir || '';
-        preparation = await latexOnline.prepareGitCompilation(req.query.git, req.query.target, 'master', command, workdir);
+        var branch = req.query.branch || 'main';
+        preparation = await latexOnline.prepareGitCompilation(req.query.git, req.query.target,branch, command, workdir);
```

Comment:
IIUC this would be a breaking change for current clients. Can we introduce a parameter only?

Reply:
sure. Changed it to master.